frontend/src/components/navbar/Navbar.tsx

```diff
@@ -109,7 +112,8 @@ const Navbar: React.FC = () => {
                 </Box>
               </Box>
               <div className={classes.sectionDesktop}>
-                <NavbarLinks></NavbarLinks>
+                <NavbarLinks loggedIn={loggedIn} />
+                <NavbarUser loggedIn={loggedIn} username={loggedIn ? data?.user?.firstName ?? "" : undefined} />
```

Comment:
```suggestion
                <NavbarUser loggedIn={loggedIn} username={data?.user?.firstName} />
```

Reply:
`firstName` can be `undefined`.

Reply:
Nevermind, you're right! Changed in 91a3ba2.